templates/digital-asset-template/frontend/pages/MyCollections.tsx

```diff
@@ -38,30 +39,32 @@ export function MyCollections() {
           </TableRow>
         </TableHeader>
         <TableBody>
-          {collections.length > 0 &&
+          {collections &&
```

Comment:
collections can be undefined/null?

Reply:
Ah, this was a vestige of the previous commit, let me fix that